localisation/00_wc_sad.csv

```diff
@@ -363,30 +363,30 @@ EVTDESC_WCCLS.10002;After a few hours of searching the wilds you notice a large
+EVTOPTA_WCCLS.10006;I have heard your words, Great Cat Spirit, and I understand.;;Ich habe Eure Worte geh�rt, gro�er Katzengeist, und ich verstehe sie.;;;;;;;;;;;x
+EVTDESC_WCCLS.10007;It remains to be seen if you are ready, young one. Even the wisest and oldest of druids are never truly ready when their ultimate time of testing comes. You have heard my words, and now you must move on. Heed what I have taught you. There will be a time when you will have your power tested. You must face your foe as the cat would - with quickness of body and with resolve of the heart. Learn from the fight, young one. Go... with my blessings.;;Es bleibt abzuwarten, ob du bereit bist, Jungchen. Selbst die weisesten und �ltesten Druiden sind nie wirklich bereit, wenn ihre Zeit der letzten Pr�fung kommt. Du hast meine Worte geh�rt, und jetzt musst du voranschreiten. H�re auf das, was ich dich gelehrt habe. Es wird eine Zeit geben, in der du deine Kraft auf die Probe stellen wirst. Du musst deinem Feind wie eine Katze gegen�bertreten - mit der Schnelligkeit deines K�rpers und mit der Entschlossenheit deines Herzens. Lerne aus dem Kampf, Jungchen. Geh... du hast meinen Segen.;;;;;;;;;;;x
 EVTOPTA_WCCLS.10007;I will stay true to the way of the Claw.;;Ich werde dem Weg der Kralle treu bleiben.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
-EVTDESC_WCCLS.10020;You start your journey to Moonglade, the sacred place of druidism. It's a tradition for young druids to learn the Bear Form from the Anicent Protector Ursoc's spirit who resides there.;;;;;;;;;;;;;x
+EVTDESC_WCCLS.10020;You start your journey to Moonglade, the sacred place of druidism. It's a tradition for young druids to learn the Bear Form from the Ancient Protector Ursoc's spirit who resides there.;;Ihr beginnt Eure Reise nach Moonglade, dem heiligen Ort des Druidismus. Es ist eine Tradition f�r junge Druiden, die B�renform vom Gro�en B�rengeist zu lernen, der dort lebt.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCCLS.10020;You start your journey to the Moonglade, the sacred place of druidism. It's a tradition for young druids to learn the Bear Form from the Ancient Protector Ursoc's spirit who resides there.;;Ihr beginnt Eure Reise nach Moonglade, dem heiligen Ort des Druidismus. Es ist eine Tradition f�r junge Druiden, die B�renform vom Gro�en B�rengeist zu lernen, der dort lebt.;;;;;;;;;;;x
```

Reply:
No, Moonglade is the name of the Province/Holding etc. It's the same as "journey to Stormwind" for example

Reply:
Looking at Wowpedia, its a little unclear. Sometimes its the Moonglade, other times its Moonglade. Either or, doesn't matter too much.